### PR TITLE
Edited FirstAndTen

### PR DESCRIPTION
**Pull request by aleeper**
@nishadhka I created this PR to track this work. I will now make a few high-level changes myself, and then make a bunch of comments on the PR with minor style things to fix. (Please wait to make more changes until I'm done commenting.)

You can also see the comments I made on this other PR, #1.

### CODE REVIEW

File: common/macrosLeeper.tex [hunk 1]
```diff
@@ -271,8 +274,10 @@
 %%%
 \newcommand{\rotationTable}[2]
 {
-  \def\dirCosTableA{#1}
-  \def\dirCosTableB{#2}
+  \def\dirCosTablea{\lowercase{#1}}
+  \def\dirCosTablea{\lowercase{#2}}
+  \def\dirCosTableA{\uppercase{#1}}
+  \def\dirCosTableB{\uppercase{#2}}
```
Comment (aleeper) [inline common/macrosLeeper.tex:280]: I see what you were trying to fix. I thought about it and I think this is the right solution.

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 2]
```diff
@@ -1,116 +1,240 @@
...
+  %position on the field.
+  \\[0.45pc]
+  Let the field be a rigid frame \basis{N} with orthogonal unit
+  vectors \uvecBasisxyz{n} oriented as shown.
+  Let point $N_o$ be fixed in \basis{N} at zero height above the field.
+  Assume the field lies in the \uvecxy{n} plane.
+  %
+  \\[0.45pc]
+  Let the camera body be a rigid frame \basis{B} with orthogonal
+  unit vectors \uvecBasisxyz{b} and a point $B_o$ fixed in \basis{B}.
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: I recently started using an `\origin` macro. So, replace instances of B_o, N_o, etc with, for example, `\origin{B}`.

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 3]
```diff
@@ -1,116 +1,240 @@
...
+  Assume the position vector from $N_o$ to $B_o$ is known to be
+  $\posvec{N_o}{B_o} \equals (50\mathrm{~yards})~\uvecx{n}
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: Can we use the siunitx package here, even though it's in yards? I think `\SI{50}{yards}` will work.

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 4]
```diff
@@ -1,116 +1,240 @@
...
+  \documentclass[11pt,letterpaper]{book}
+  \input{handout_helpers.tex}   % Found via TEXINPUTS environment variable.
+  \isolatedBuildHeader{Vector Equations and Geometry}
+                      {Cameras and Robotics in the NFL}
 }
+%%%
+%%%
+%%%
+\newcommand\MyLBrace[2]{%
+  \left.\rule{0pt}{#1}\right\}\text{#2}}
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: Can't define newcommands in these problem files, or they won't work when I `\input` a bunch of them into a single document.

Also, I don't think we need this.

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 5]
```diff
@@ -1,116 +1,240 @@
...
+       dot w/ \uvecx{n}: & x & $ 50 \plus \lambda \uvecx{b} \DotProduct \uvecx{n} $
+       \\[1.0pc]
+       dot w/ \uvecy{n}: & y & $ 0 \plus \lambda \uvecx{b} \DotProduct \uvecy{n} $
+       \\[1.0pc]
+       dot w/ \uvecz{n}: & o & $ 20 \plus \lambda \uvecx{b} \DotProduct \uvecz{n} $
+       \\[1.0pc]
+       \end{tabular}
+       &
+       $\begin{array}{l}
+       \MyLBrace{8.9ex}{scalar system w/ 3 equations, 3 unknowns x,y,$\lambda$}
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: This does a good job of matching what I drew on the page, but I think it's better for typesetting in this case to just put this on the next line, e.g. "The 3 equations above form a scalar system of equations with 3 unknowns."

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 6]
```diff
@@ -1,116 +1,240 @@
...
+  \flushright
+  \vspace*{0pt}
+  \includegraphics[width=7cm]{camera_pan_tilt.jpg}
+  \end{minipage}
+\Solution{}{0.99\linewidth}{
+  \begin{minipage}{0.34\linewidth}
+    \minipageTopAnchor
+    \center
+    \rotationTable{N}{A}
+    {\cos\theta}{\minus\sin\theta}{0}
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: hmm... it looks weird with so much space after the minus sign. Maybe just use `-` for these.

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 7]
```diff
@@ -1,116 +1,240 @@
...
+       \\[1.0pc]
+       \end{tabular}
+       &
+       $\begin{array}{l}
+       \MyLBrace{8.9ex}{scalar system w/ 3 equations, 3 unknowns x,y,$\lambda$}
+       \end{array}$
+       \end{tabular}
+       \end{minipage}
+       \\[0.45pc]
+       }
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: The `\end{}` macros and the closing `}` should match the indentation level of the corresponding `\begin{}` and `\Solution` above.

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 8]
```diff
@@ -1,116 +1,240 @@
...
+    \includegraphicsAB[width=0.66\linewidth]
+      {football1.png}
+      {football.png}
+  \end{center}
 \end{minipage}
-
+\Solution{}{0.99\linewidth}{
+  \begin{minipage}{0.65\linewidth}
+     \begin{tabular}{@{ }l@{}r@{ \equals[\;] }ll}
+       &\posvec{N_o}{Q}& $x\uvecx{n} \plus y\uvecy{n} \plus o\uvecz{n}$
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: space after &.
Also, it looks like you put an `o` in a bunch of places where it was supposed to be `0`. :)

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 9]
```diff
@@ -1,116 +1,240 @@
...
+       \\[1.0pc]
+       &\posvec{N_o}{B_o}& $50\uvecx{n} \plus 0\uvecy{n} \plus 20\uvecz{n}$
+       \\[1.0pc]
+       &\posvec{B_o}{Q} & $\lambda\uvecx{b}$
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: I like putting a non-breaking space before unit vectors, e.g. `\lambda~\uvecx{b}`. Please put ~ before all the unit vectors.

File: material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex [hunk 10]
```diff
@@ -1,116 +1,240 @@
...
+  Then \textbf{\underline{explain}} how to obtain the rotation
+  table relating frame \basis{N} and \basis{B}
+  (don't actually compute it).
+  \end{minipage}
+  \hfill
+  \begin{minipage}[t]{0.37\columnwidth}
+  \flushright
+  \vspace*{0pt}
+  \includegraphics[width=7cm]{camera_pan_tilt.jpg}
+  \end{minipage}
```
Comment (aleeper) [inline material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex]: Each time after \begin{}, or after starting a multi-line macro, or after starting a multi-line math statement, the code should indent an additional two spaces. Then when you reach \end{}, or when the multi-line macro ends, or when the multi-line math statement ends, you un-indent two spaces.

**Comment by aleeper** [comment]
I'm done with a round of comments. Please make sure you do `git pull` to get my changes to your branch, and then make additional commits and push.

It's important to me that the latex code be formatted so that it's easy to see where things begin and end. Putting the necessary whitespace and indentation is an important part of that. You might look at `material/problems/VectorBasics/ResultantForce1` or `material/problems/VectorBasics/SingleBasisMoment`, which are recent problems where I really took care to line things up in the code itself so it's easy to read and modify.

File: common/macrosLeeper.tex (hunk 1 above)
Comment (nishadhka) [inline common/macrosLeeper.tex:280]: This is better, found some issue, unit vectors are repeating. It is an typo and changed then.

**Comment by nishadhka** [comment]
Changes were made regarding last comments, @aleeper

**Comment by aleeper** [comment]
@nishadhka 
I "squashed" all the commits to just 2 commits: one for the change to the rotationTable macro, and one for the actual changes to the FirstAndTen problem. This way the git history is more concise (though, it's totally fine to have small incremental commits during development).

I'm going to merge this PR as-is for now. Please consider the "transcrib-nish" branch done. Any future work will happen on a new branch.

**Pull request merged**